examples/hello.rs

```diff
@@ -16,6 +16,6 @@ fn hello(_: Request, res: Response) {
 
 fn main() {
     let _listening = hyper::Server::http(hello)
-        .listen(IpAddr::new_v4(127, 0, 0, 1), 3000).unwrap();
+        .listen_threads(IpAddr::new_v4(127, 0, 0, 1), 3000, 10).unwrap();
```

Comment:
Oh good catch. I was altering numbers when testing performance improvement for the server.